### ISSUE TITLE
Should add quick add and commit (Do both in one action)

### ISSUE THREAD

**Issue by brycefranzen** [SublimeGit/SublimeGit#211]

**Comment by pingshunhuangalex** [comment]
Isn't that just equivalent to quick commit?

**Comment by brycefranzen** [comment]
No. Quick commit alone will show that the working directory is clean and that there is nothing to commit unless a quick add has been done first.

**Comment by pingshunhuangalex** [comment]
Mmm, but I never use add before. Whenever I finish tweaking my code, I just do a quick commit and push, and it seems to be all fine. Maybe I got some misunderstanding there?

**Comment by pingshunhuangalex** [comment]
Right, so you just want to add and commit a new file, not commit a changed file.

**Comment by brycefranzen** [comment]
I would like a single function that when called will add all unstated files and then prompt for a commit message. Upon being entered, should commit all files.  Currently I find myself frequently running “quick add” upon which I type “+” for all files, followed by “quick commit”.  It would be nice to have one action that does all of this.

**Comment by pingshunhuangalex** [comment]
Right gotcha now. I'm expecting a similar feature => quick add + quick commit + push

I think I'll just borrow your thread here :)

**Comment by brycefranzen** [comment]
cool.  Yeah, both features would be really nice.

**Comment by rsiebeling** [comment]
It would be great if it is possbile to do the following commands in one
git add -A
git commit
git push

That way all changes are added, committed and pushed to the server

**Comment by radlinsky** [comment]
> It would be great if it is possbile to do the following commands in one
> git add -A
> git commit
> git push
> 
> That way all changes are added, committed and pushed to the server

Agreed!

Updates?
